client/src/extension.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind,
+} from 'vscode-languageclient'
+import { ExtensionContext, workspace } from 'vscode'
+import path from 'path'
+
+let client: LanguageClient
+
+export async function activate(context: ExtensionContext) {
+  // The server is implemented in node
+  const serverModule = context.asAbsolutePath(
+    path.join('server', 'out', 'server.js'),
+  )
+  // The debug options for the server
+  // --inspect=6009: runs the server in Node's Inspector mode so VS Code can attach to the server for debugging
+  const debugOptions = { execArgv: ['--nolazy', '--inspect=6009'] }
```

Comment:
Do we need to run the server in debug mode?

Reply:
yep